pylossless/pipeline.py

```diff
@@ -1037,13 +1037,19 @@ def save(self, derivatives_path, overwrite=False):
             path of the derivatives folder to save the file to.
         overwrite : bool (default False)
             whether to overwrite existing files with the same name.
+        format : str (default "EDF")
+            The format to use for saving the raw data. Can be 'auto',
+            'FIF', 'EDF', 'BrainVision', 'EEGLAB'.
```

Comment:
```suggestion
            ``"FIF"``, ``"EDF"``, ``"BrainVision"``, ``"EEGLAB"``.
```

nitpick. We try to wrap parameter options in backticks so that they render as code in the online Sphinx documentation. We also prefer double quotes to single quotes so that we follow [black](https://github.com/psf/black) formatter conventions).